client/blocks/importer/wordpress/upgrade-plan/upgrade-plan-details.tsx

```diff
@@ -46,10 +45,10 @@ export const UpgradePlanDetails = ( props: Props ) => {
 	}, [ plan ] );
 
 	return (
-		<div className={ classnames( 'import__upgrade-plan-details' ) }>
```

Comment:
Here and in some other cases of this file, I just clean up these single classnames where the function is not needed.